nomad/structs/structs.go

```diff
@@ -1445,6 +1445,25 @@ const (
 	DefaultKillTimeout = 5 * time.Second
 )
 
+// LogConfig provides configuration for log rotation
+type LogConfig struct {
+	MaxFiles      int `mapstructure:"max_files"`
+	MaxFileSizeMB int `mapstructure:"max_file_size"`
+}
+
+// MeetsMinResources returns an error if the log config specified are less than
```

Comment:
Name `Validate` and only check that max file size and max files is greater than or equal 0